cmd/snap-update-ns/utils.go

```diff
@@ -349,19 +349,19 @@ func planWritableMimic(dir string) ([]*Change, error) {
 	}
 	for _, fi := range entries {
 		ch := &Change{Action: Mount, Entry: osutil.MountEntry{
-			Name:    filepath.Join(safeKeepingDir, fi.Name()),
-			Dir:     filepath.Join(dir, fi.Name()),
-			Options: []string{"bind"},
```

Comment:
Hmm, so options was set for symlink previously. Good catch (even if it went unused).

Reply:
No, because the symlink case was over-writing options. This is here so that we can have either bind or rbind.